File: 2st/Riyeon/삼각형의 완성조건 (2).py
def solution(sides):
    answer = 0
    maxs, mins = max(sides), min(sides)
    n = 1
    while n < sum(sides):
        if mins + n > maxs:
            answer += 1
        elif maxs >= n and n > sum(sides):
            answer += 1
        n += 1
    return answer

File: 3st/Juhee/삼각형의 완성조건 (2).java
class Solution {
    public int solution(int[] sides) {
        int a =0;
        int b =0;
        int c =0;
        if(sides[0]>=sides[1]){
            b = sides[0];
            c = sides[1];
        } 
        else{
            b = sides[1];
            c = sides[0];
        }
        int answer=0;
        a = b;
        while(a<b+c){
            answer++;
            a++;
        }
        a = b-1;
        while(b<c+a){
            answer++;
            a--;
        }
        return answer;
    }
}